source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st

```diff
@@ -67,7 +65,7 @@ PooledRDBMSSessionProvider >> createNewSession [
 	session := GlorpSession new.
 	session accessor: self createDatabaseAccessor.
 	session system: descriptorSystem.
-	session loginIfError: [ :error | DatabaseLoginFailed signal: error description ].
+	self ensureSuccessfulLoggingToDatabaseWith: session.
```

Comment:
I'm pretty sure you don't want to use `logging` here, but `login` or if you want to use the past continuous is `logging in`